### DISCUSSION TITLE
Any way to create modal over a component?

### DISCUSSION THREAD

**Discussion by bombillazo** [ant-design/ant-design#43001] [Q&A] [answered]
Hello, let's say i want to disable the calendar component and show a modal message, how can that be achieved with Ant Design?

**Comment by zombieJ** [comment]
I'm not sure get your point. Calendar can only disable date cell. But you can use style to handle this:
https://codesandbox.io/s/ji-ben-antd-5-6-1-forked-x3w9x9?file=/demo.tsx

**Reply by bombillazo** [reply]
Hey, my question I guess is more about if I can choose a specific area or component over which to render the modal instead of the whole screen, for example in the calendar car, render the modal only inside the space occupied by the calendar.

**Reply by zombieJ** [reply, marked as answer]
It's little strange for this requirement. If so, you can use `getContainer` to config modal render position and add `style` to make it position absolute.

**Reply by bombillazo** [reply]
I will try it out, thanks!
